src/main/java/com/currencycloud/client/CurrencyCloud.java

```diff
@@ -718,6 +719,7 @@ Ibans findIbans(
     /** Retrieve IBAN of Sub-Account(s) */
     @GET
     @Path("ibans/subaccounts/{id}")
+    @Deprecated
```

Comment:
Should the reason/alternative to the deprecation be documented in the javadoc?

Reply:
Reason and alternative are specified in the implementation class (CurrencyCloudClient). Interface (CurrencyCloud) is marked for completeness and consistency